datadog/templates/rbac.yaml

```diff
@@ -0,0 +1,55 @@
+{{- if .Values.rbac.enabled }}
+
+---
```

Comment:
I our cluster `Tiller` have the right to create a `RBAC` ? :astonished:

Reply:
Only in the protected kube-system namespace)